extras/indexing/src/test/java/org/apache/rya/indexing/mongo/MongoDbSmartUriTest.java

```diff
@@ -74,14 +71,13 @@
 
 import com.beust.jcommander.internal.Lists;
 import com.google.common.collect.ImmutableSet;
-import com.mongodb.MongoClient;
 
 import info.aduna.iteration.CloseableIteration;
 
 /**
  * Tests for MongoDB based Smart URI.
  */
-public class MongoDbSmartUriTest {
+public class MongoDbSmartUriTest extends MongoITBase {
```

Comment:
this is an odd mix of integration tests and unit tests >..> should probably be split up into a Test and an IT